package.json

```diff
@@ -7,12 +7,25 @@
   "author": "Akshay S Dinesh <asdofindia@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "jspm bundle src/ascii-unicode-converter.js dist/bundle.js --skip-source-maps"
+    "run-server": "node_modules/http-server/bin/http-server --cors -p 9000",
+    "postinstall": "cp node_modules/siimple/dist/*.min.css src/assets/ && cp node_modules/siimple-colors/dist/*.min.css src/assets/",
+    "build:css": "cp -R src/assets/ dist/",
```

Comment:
postinstall ensures all css required for project ends up in assets. 

Build just depends on computation (jspm and whatever) and its entire input comes from src folder and has no dependency on node_modules.

Thus the only time node_modules (or whatever it is) comes into play is during postinstall, once. The dependencies for build is minimized to src folder. That's how I see it. If you need me to do it another way I'll gladly change it.